src/styles/card.scss

```
@@ -0,0 +1,105 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-inline: 19px;
+    margin-top: 32px;
+  }
+
+  &__discription {
+    color: $main-text-color;
+    margin-inline: $default-margin;
+    margin-bottom: 0;
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__code {
+    margin-inline: $default-margin;
+    margin-top: 4px;
+    margin-bottom: 0;
+    color: $side-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
```

Default value, so you can just remove it everywhere
```suggestion
    line-height: 14px;
```